examples/player/simid_player.html

```diff
@@ -116,7 +116,17 @@
     }
 
     function switchCreative(value) {
-      document.getElementById('creative_url').value = '../creatives/' + value;
+      const creative_url = document.getElementById('creative_url').value = '../creatives/' + value;
+
+      if (creative_url.includes("nonlinear")){
+        document.getElementById("nonlinear_ad").checked = true;
+        document.getElementById("linear_ad").checked = false;
+      }
+      else {
+        document.getElementById("linear_ad").checked = true;
+        document.getElementById("nonlinear_ad").checked = false;
```

Comment:
moreover, i don't think that this setup works in making the creative dimension and duration field pop up automatically since the function that does that is called onclick.

Reply:
N/A after merge